Tests/Unit/MSFT_xWebsite.Tests.ps1

```diff
@@ -1236,19 +1236,19 @@ try
 
                 $MockBindingInfo = @(
                     New-CimInstance -ClassName MSFT_xWebBindingInformation -Namespace root/microsoft/Windows/DesiredStateConfiguration -Property @{
-                        Protocol              = 'http'
+                        Protocol              = 'https'
                         IPAddress             = '*'
-                        Port                  = 8080
+                        Port                  = 443
```

Comment:
None of the changes to the test file seem necessary to fix these issues, if you would like to make the tests "more realisitc" could you please submit it as a seperate PR?